api-compact.yaml

```diff
@@ -85,9 +91,11 @@ x-amazon-apigateway-integration: &apigateway_integration
   type: "http"
   requestParameters:
     integration.request.header.Authorization: "method.request.header.Authorization"
+    integration.request.header.Host: "${stageVariables.enterprise_host}"
```

Comment:
According to the API Gateway [known issues document](http://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-known-issues.html), the host mapping is unavailable only when the system is configured for HTTP proxy mode, so I think we're in the clear?  From what I've read in that doc, the name of the Host header may actually be `x-amzn-Remapped-Host`